### DISCUSSION TITLE
Reed contact and mqtt

### DISCUSSION THREAD

**Discussion by baba2k** [jomjol/AI-on-the-edge-device#1325] [Q&A]
Hi,

my gasmeter has also a reed contact. is it possible to use the io pins to expose these values via mqtt with the same esp32?

Thanks!

**Comment by caco3** [comment]
You could try to use it as following (red = Expert Mode):
![grafik](https://user-images.githubusercontent.com/1783586/201199523-ba244d25-fc11-4d4a-9e03-b4e7301e8d61.png)

I have not tested it myself!
